e2e/app/element/base-element.ts

```diff
@@ -223,10 +216,11 @@ export default class BaseElement {
    * Clear value in textbox.
    */
   async clear(options: ClickOptions = { clickCount: 3 }): Promise<void> {
-    const elemt = await this.asElementHandle();
-    await elemt.focus();
-    await elemt.hover();
-    await elemt.click(options);
+    const element = await this.asElementHandle();
+    await element.click();
+    await this.page.waitForTimeout(500);
+    await element.click(options);
+    await this.page.waitForTimeout(500);
```

Comment:
Rework `clear` function to make it less flaky. I watched one test playback and saw `clear` function was not running as expected.

Reply:
Can you add a comment for any "sleep 500ms" like this. These slow down the tests, can indicate or introduce underlying flakiness, and are hard to remove later.

Reply:
I'm choosing reliability over speed. A short sleep ensures mouse cursor is blinking and field is ready for input after clicks. Making a video to show the problem before change.